testsuite/features/step_definitions/api_common.rb

```diff
@@ -269,6 +269,15 @@
 
   # Get the list of child channels for this base channel
   child_channels = $api_test.channel.software.list_child_channels(base_channel_label)
+
+  # filter out wrong child channels for SLE Micro 5.5 as normal Minion
+  if client.include? 'slemicro55'
+    child_channels.reject! { |channel| channel.include? 'suse-manager-proxy-5.0-pool-x86_64' }
+    child_channels.reject! { |channel| channel.include? 'suse-manager-proxy-5.0-updates-x86_64' }
+    child_channels.reject! { |channel| channel.include? 'suse-manager-retail-branch-server-5.0-pool-x86_64' }
+    child_channels.reject! { |channel| channel.include? 'suse-manager-retail-branch-server-5.0-updates-x86_64' }
+  end
```

Comment:
```suggestion
  if client.include? 'slemicro55'
    child_channels.reject! { |channel| channel.include? 'suse-manager' }
  end
```

Reply:
This would indeed be a nice simplification. However, it could cause issues in the future with similar channel names. The way I did it, we clearly see, which channels should be filtered out.

Reply:
OK, let's follow that approach.
It will require more maintnenance in the future, but it's more concise.